core/src/components/checkbox/checkbox.scss

```diff
@@ -72,7 +72,9 @@
   flex-grow: 1;
 
   align-items: center;
+  justify-content: space-between;
 
+  width: 100%;
```

Comment:
This is required so that long labels will ellipsis instead of running off the screen.